erpnext/regional/india/utils.py

```diff
@@ -152,8 +152,11 @@ def get_itemised_tax_breakup_data(doc, account_wise=False):
 def set_place_of_supply(doc, method=None):
 	doc.place_of_supply = get_place_of_supply(doc, doc.doctype)
 
-def validate_document_name(doc, method=None):
+def validate_document_name(doc, method=None, validation_test=False):
 	"""Validate GST invoice number requirements."""
+	if frappe.flags.in_test and not validation_test:
```

Comment:
This isn't a good fix. The entire purpose of the entire purpose of tests is to fail on validations so that the problematic code can be fixed.

I'm working on a PR to set different `naming_series` options for Indian companies.

Reply:
PR Link: https://github.com/frappe/erpnext/pull/25128